tutorials/users/ide/ide.md

```diff
@@ -1 +1,9 @@
 # IDE
```

Comment:
Change this to "Interactive Development Environment"

(don't use acronyms before they've been introduced in text)

Reply:
Changed to use full wording.